WebDevStudios/wd_s
# Updates to Tailwind 2.0

### DESCRIPTION
Updates to Tailwind 2.0!

More specifically, Tailwind 2.0.1 compatibility mode. Why? Details here: https://tailwindcss.com/docs/installation#post-css-7-compatibility-build

>As of v2.0, Tailwind CSS depends on PostCSS 8. Because PostCSS 8 is only a few months old, many other tools in the ecosystem haven't updated yet, which means you might see an error like this in your terminal after installing Tailwind and trying to compile your CSS:
> `Error: PostCSS plugin tailwindcss requires PostCSS 8.`

This happened for us due to other tools in our system not being PostCSS 8-ready. Tailwind 2.0 compatibility mode solves that issue.

This also adds the `darkMode: 'class',` option to the Tailwind config file to prep for future dark mode implementations.

### STEPS TO VERIFY
Check out the branch and run `npm i`.

Right now this doesn't do anything other than bump us up to 2.0.1. None of our markup/Sass has changed, but we're now ready to use any of the new goodness from Tailwind 2.0.

### DOCUMENTATION
We'll want to make a small update here to add in the details around dark mode.: https://github.com/WebDevStudios/wd_s/wiki/Tailwind
